apps/server/src/lib/ts-helpers.ts

```diff
@@ -0,0 +1,12 @@
+export type Primitive = string | boolean | number;
```

Comment:
This needs to be shared between FE and BE, I'll create a `ts-helpers` library soon